src/components/Dashboard/Navbars/PsychologistNavbar.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styles from '../Dashboard.module.css';
+import logo from '../../../assets/logo.png';
+import btnMenu from '../../../assets/menuIcon.png';
+
+function PshycologistNavbar() {
+	const [state, setState] = useState(false);
+
+	const asideOpenAction = () => {
+		if (!state) {
+			setState(true);
+		} else {
+			setState(false);
+		}
+	};
+	const Pshycologist = (
```

Comment:
is this a constant?
dond use PascalCase on constants
check the naming convention [https://github.com/lalodiazdiaz/Emotions/wiki/Code-Review](url)